### ISSUE TITLE
Does this work with minifiers?

### ISSUE THREAD

**Issue by lostfictions** [nasser/ajeeb-ecs#4]
Hey, thanks for sharing this code! I like how ergonomic and simple it seems to make defining components and systems. At a glance it doesn't seem like the way systems are defined would be compatible with minifiers, though -- as soon as function parameters are mangled things will break. Is that correct?

**Comment by lostfictions** [comment]
Ah, took a look and saw that the `onX` methods are each just sugar for calling the respective `addXSystem` methods with explicit dependencies. This does seem like a thing that could catch folks unawares if they're trying this out, though...

**Comment by nasser** [comment]
yes, the on* methods are just sugar (fairly aggressive sugar at that!). this library was originally built for electron where the size of code is vanishingly small compared everything else, so I hadn't considered minifiers. I think a note in the documentation is a good idea.

R
